### ISSUE TITLE
Generate .txt file segments in parallel using tortoise_tts.py?

### ISSUE THREAD

**Issue by cboRD181** [neonbjb/tortoise-tts#198]
Sorry to be creating another issue.  

I've notice my RAM utilization floats around ~3GB when running read.py or tortoise_tts.py in colab, so was wondering if it was possible to run text segment generations in parallel to more fully utilize the resources? 

I tried setting the --batch-size flag but still only got one segment outputted at a time, and the same RAM use (though the autoregressive sample generation step took longer).  Maybe I misunderstand that flag's purpose, but is what I'm asking possible?  It'd make longform generation a lot more tractable.

**Comment by neonbjb** [comment]
I'm assuming you mean standard memory, not VRAM. More fully utilizing this won't improve performance. The `batch-size` flag (should) impact VRAM use which won't show up in collab, but should improve performance a bit.

**Comment by cboRD181** [comment]
Ah, I see. Thanks for clearing up my confusion.
